data_model/src/lib.rs

```diff
@@ -2246,6 +2246,8 @@ pub mod uri {
     pub const BLOCK_SYNC: &str = "block";
```

Comment:
I would recommend renaming this to `sync`

Reply:
No problem, though personally I'd call it `block/sync`, both because we might have other information about the block later down the line, but also because it's clearer. There are many things that Iroha _could_ sync, especially when runtime upgrades are enacted.

Reply:
what about `BLOCK_SYNC = sync/blocks` and `BLOCKS_STREAM = stream/blocks`?

Reply:
Acceptable, but we use a domain-first structure, right?

Reply:
but what do you consider a domain here?

and is the alternative here `BLOCK_SYNC = blocks/sync` and `BLOCKS_STREAM = blocks/stream`?
I kinda prefer the previous naming convention but idk. Is there some other alternative?

Reply:
`block/sync` and `block/stream` sounds good to me.

Reply:
ok